doc/source/configuration/vault.rst

```diff
@@ -296,6 +296,7 @@ Configure Barbican
       [vault_plugin]
       vault_url = https://{{ kolla_internal_vip_address }}:8200
       use_ssl = True
+      ssl_ca_crt_file = /etc/ssl/certs/ca-bundle.trust.crt
```

Comment:
It would be better to reference `openstack_cacert` since this is distro-specific. You'll need to wrap in `raw` since its a Kolla variable.

Reply:
Done :)